site/gatsby-site/src/templates/cite.js

```diff
@@ -564,16 +572,23 @@ export const query = graphql`
         date_published
         report_number
         title
+        description
         url
         image_url
         cloudinary_id
         source_domain
         mongodb_id
         text
+        plain_text
```

Comment:
Adding these fields almost doubles the amount of data required to build a citation page, which, multiplied by the number of incidents * number of languages, is a lot more data being used.

I wonder if build time changed much after this change 🤔

Reply:
I confirmed that adding these extra fields doesn't affect the build time of the citation page. Anyways, I removed them since they are not needed anymore.